collector/container/Dockerfile.ubi

```diff
@@ -85,32 +87,24 @@ COPY --from=builder ${BUILD_DIR}/src/container/scripts /
 # Switch shell to bash with pipefall to avoid hadolint warning DL4006
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
-# hadolint ignore=DL3041
+# hadolint ignore=DL3041,SC2046
 RUN mv /collector-wrapper.sh /usr/local/bin/ && \
     chmod 700 bootstrap.sh && \
     echo '/usr/local/lib' > /etc/ld.so.conf.d/usrlocallib.conf && \
     ldconfig && \
     mkdir /kernel-modules && \
-    microdnf upgrade -y && \
-    microdnf install -y \
-      --enablerepo=codeready-builder-for-rhel-8-x86_64-rpms \
-      --enablerepo=amq-clients-2-for-rhel-8-x86_64-rpms \
-      --enablerepo=rhocp-4.6-for-rhel-8-x86_64-rpms \
-      gzip \
-      kmod \
-      jsoncpp \
-      tbb \
-      jq \
-      c-ares && \
+    microdnf upgrade && \
+    microdnf -y install c-ares kmod tbb && \
+    microdnf -y install jsoncpp --enablerepo=amq-clients-2-for-rhel-8-x86_64-rpms && \
+    microdnf -y install jq --enablerepo=rhocp-4.6-for-rhel-8-x86_64-rpms && \
     if [ "$USE_VALGRIND" = "true" ]; then \
-      microdnf install -y valgrind; \
+      microdnf -y install valgrind; \
     fi && \
     if [ "$ADDRESS_SANITIZER" = "true" ]; then \
-        microdnf -y install libasan; \
+      microdnf -y install libasan; \
     fi && \
     microdnf clean all && \
-    rpm --query --all 'curl' '*rpm*' '*dnf*' '*libsolv*' '*hawkey*' 'yum*' |\
-      xargs rpm -e --nodeps && \
+    rpm -e --nodeps $(rpm -qa curl '*dnf' '*solv*' '*rpm*') && \
```

Comment:
The SC2046 hadolint exception is for this unquoted shell evaluation. `xargs` isn't installed in ubi-minimal by default and I didn't want to add it just for this line.